### PR TITLE
remove numpy, scikit-learn, matplotlib, and pandas posts 

### PR DESCRIPTION
**Pull request by jdamiba**
closes https://github.com/plotly/documentation/issues/1529

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
This can't be merged until we deploy the recent changes to prod. 

Also: could we do a full scrub of all the includes etc and remove anything specific to numpy/scikit-learn?

**Comment by nicolaskruchten** [comment]
(redirects are now deployed in prod)

**Comment by nicolaskruchten** [comment]
let's leave this open until we finish the pandas/matplotlib redirects are in place and roll those removals into this PR

**Comment by nicolaskruchten** [comment]
(and the `ipython-notebook` migration!)

**Comment by jdamiba** [comment]
@nicolas I believe that this PR should be reviewed in its current state and the work to move the `ipython-notebooks` content into the `python` content should occur in a separate PR.

The rationale for this is that these set of changes all depended on `streambed` configuration changes while the `ipython-notebooks` work does not. 

This PR removes `numpy`, `scikit-learn`, `matplotlib`, and `pandas` posts from this repo, along with all references to those languages in Liquid conditionals and Jekyll configuration files.

File: _includes/layouts/lang-navigation.html [hunk 1]
```diff
@@ -11,8 +11,6 @@
 		{%- assign pyclass = "current" -%}
 	{%- elsif page.language == "ipython-notebooks" -%}
 		{%- assign ipyclass = "current" -%}
-	{%- elsif page.language == "pandas" -%}
-		{%- assign pdclass = "current" -%}
```
Comment (nicolaskruchten) [inline _includes/layouts/lang-navigation.html]: does it make sense to remove `pdclass` and `matplotclass` and anywhere that refers to them?

**Comment by nicolaskruchten** [comment]
💃

**Pull request merged**